src/app/shell/personal-cabinet/provider/create-provider/create-provider.component.ts

```diff
@@ -161,4 +180,23 @@ export class CreateProviderComponent extends CreateFormComponent implements OnIn
       }
     });
   }
+
+  logout(){
```

Comment:
let's rename this function to "onCancel" because we not really want o logout, we want cancel the registration